### ISSUE TITLE
[Bug]: X-Mouse Button Control can't click at "No change" button

### ISSUE THREAD

**Issue by max20091** [bottlesdevs/programs#86]

**Comment by max20091** [comment]
> I don't think I understand much about the issue but I guess it's a Windows program

Yes, it's a Windows program that does swap mouse buttons & doing macros.
Inside the program there's is a list of buttons to swap, default parameter is "No Change" and when I press the "No Change", it doesn't making a list properly.
![image](https://user-images.githubusercontent.com/12845078/173789512-a3401b04-919f-4a3a-b3be-96aeefcfc0ac.png)

**Comment by mirkobrombin** [comment]
Try launching it with the terminal and attach output here

**Comment by max20091** [comment]
`wineserver: using server-side synchronization.
002c:err:wineboot:process_run_key Error running cmd L"C:\\windows\\system32\\winemenubuilder.exe -r" (2).
MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0
`
Doesn't seems to throw any bugs. Well, you can download the program for free and take a look at here: https://www.highrez.co.uk/downloads/xmousebuttoncontrol.htm
